best-practice-authorization.md

```diff
@@ -18,7 +18,7 @@ implement specific NMOS Interface Specifications.
 
 ### IS-04 - Discovery and Registration
 
-When registering resources with an instance of an [IS-04 Registry], the Registry MUST register the Client ID
+When registering resources with an instance of an [IS-04 Registry][], the Registry MUST register the Client ID
```

Comment:
That doesn't look right?

Reply:
\[Discussed on slack\]... it does look odd but does work and makes the line pass remark-lint-no-shortcut-reference-link.

Reply:
Yep, Jekyll is using kramdown which supports this https://kramdown.gettalong.org/syntax.html#reference-links